pages/linux/journalctl.md

```diff
@@ -23,6 +19,10 @@
 
 `journalctl -u {{unit}}`
 
+- Show logs from last unit start:
```

Comment:
Perhaps this will read better?
```suggestion
- Show all logs since the last time a given unit started:
```

Is this all logs from a given unit, or all lots from all units since the given unit started?

Reply:
The command shows journal entries from the most recent start for a given unit.